clang/test/Driver/aarch64-v86a.c

```diff
@@ -13,3 +13,57 @@
 // RUN: %clang --target=aarch64_be -mbig-endian -march=armv8.6a -### -c %s 2>&1 | FileCheck -check-prefix=GENERICV86A-BE %s
 // RUN: %clang --target=aarch64_be -mbig-endian -march=armv8.6-a -### -c %s 2>&1 | FileCheck -check-prefix=GENERICV86A-BE %s
 // GENERICV86A-BE: "-cc1"{{.*}} "-triple" "aarch64_be{{.*}}" "-target-cpu" "generic" "-target-feature" "+v8.6a"{{.*}} "-target-feature" "+neon"
+
+// ===== Architecture extensions =====
+
+// RUN: %clang -target aarch64 -march=armv8.6-a --print-enabled-extensions 2>&1 | FileCheck -check-prefix=ARCH-EXTENSION --implicit-check-not FEAT_ %s
```

Comment:
What is the implicit-check-not doing here? Or rather, what possible bug is it making sure we don't miss?

Is it that in the information we list the names as "AMUv1" for example, so if we forgot to add a name, we'd output just "FEAT_" ?

Reply:
It ensures that every emitted line is covered by a CHECK.

Reply:
Ok got it, the CHECK consumes the line so if there was anything left matching FEAT_, something was forgotten. Thanks.